src/usr/local/www/diag_reboot.php

```diff
@@ -43,6 +43,8 @@
 $guiretry = 20;		// Seconds to try again if $guitimeout was not long enough
 
 $pgtitle = array(gettext("Diagnostics"), gettext("Reboot"));
+$platform = system_identify_specific_platform();
+$no_options = array('SG-1100', 'ROGUE-1', 'uFW');
```

Comment:
This isn't very future-proof, though it is a similar method to that used by `/etc/rc.initial.reboot`. For example it's already out of date since it's missing upcoming models listed in the factory repository. It is probably better to check if the value of `get_single_sysctl('hw.machine')` or `php_uname('m')` is `arm`, and then in the future if some models work we can whitelist the ones known to work.

Reply:
done